helloworld-mutual-ssl/configure-client-certs.cli

```diff
@@ -0,0 +1,18 @@
+# Configure a key-store in the Elytron subsystem. The path to the keystore file doesn’t actually have to exist yet.
```

Comment:
Just minor, but configure-client-certs.cli could be renamed to just configure-client-cert.cli.

Reply:
FIxed! 
Thank you!